### PR TITLE
add new models

### PR DESCRIPTION
**Pull request by pedropaulofb**
add more than 20 new models and update mgic

### CODE REVIEW

File: models/heirbrant2023boekbazaar/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+contributor:
+ - https://www.linkedin.com/in/joachim-heirbrant-532aa1233/
+ - https://dblp.org/pid/251/5991
+ - https://dblp.org/pid/95/6356
+keyword:
+ - digital platform ontology
+ - digital platform
+ - marketplace
+theme: Class H - Social Sciences
+editorialNote: Ontology created during an Ontology Engineering lecture by students who wanted to remain anonymous. The ontology was build in a different editor tool and imported to Visual Paradigm, the current diagrammation may not exacly match the original one.
```
Comment (tgoprince) [inline models/heirbrant2023boekbazaar/metadata.yaml:14]: This note seems wrong. The author is identified by his/her thesis.

File: models/heirbrant2023boekbazaar/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ontologyType:
+ - domain
+ - application
+language: en
+designedForTask:
+ - conceptual clarification
+ - information retrieval
+ - software engineering
+context:
+ - industry
```
Comment (tgoprince) [inline models/heirbrant2023boekbazaar/metadata.yaml:24]: I would classify an ontology developed as part of a master thesis as research only.

File: models/heirbrant2023boekbazaar/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+title: Boekbazaar Ontology
+acronym:
+issued: 2023
+modified:
+contributor:
+ - https://www.linkedin.com/in/joachim-heirbrant-532aa1233/
+ - https://dblp.org/pid/251/5991
+ - https://dblp.org/pid/95/6356
+keyword:
+ - digital platform ontology
```
Comment (tgoprince) [inline models/heirbrant2023boekbazaar/metadata.yaml:10]: We use this field to identify domains, not for any keywords that may apply to the model.

File: models/machacova2023gym/original-diagrams/posilovna.png
Comment (tgoprince) [inline models/machacova2023gym/original-diagrams/posilovna.png:1]: Please check the diagram name.

File: models/mereology-examples2013/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+keyword:
+ - ontouml
+ - mereology
+theme: Class Q - Science
+editorialNote: Ontology presented in a lecture. The ontology was build in a different editor tool and imported to Visual Paradigm, the current diagrammation may not exacly match the original one.
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - learning
```
Comment (tgoprince) [inline models/mereology-examples2013/metadata.yaml:15]: I do not think this applies. See my previous comment about the use of the `learning` task.

File: models/mereology-examples2013/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+title: Example of OntoUML Mereological Relations
+acronym:
+issued: 2013
+modified:
+contributor:
```
Comment (tgoprince) [inline models/mereology-examples2013/metadata.yaml:5]: The lecturer chose to remain anonymous?

File: models/mereology-examples2013/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+keyword:
+ - ontouml
+ - mereology
+theme: Class Q - Science
+editorialNote: Ontology presented in a lecture. The ontology was build in a different editor tool and imported to Visual Paradigm, the current diagrammation may not exacly match the original one.
+ontologyType:
+ - domain
```
Comment (tgoprince) [inline models/mereology-examples2013/metadata.yaml:12]: Another pattern collection.

**Review by tgoprince — CHANGES_REQUESTED** [review]
See comments.

File: models/falbo2017patterns/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+title: Example of OntoUML Patterns
+acronym:
+issued: 2017
+modified:
+contributor:
+ - https://dblp.org/pid/229/4259
+keyword:
+ - ontouml
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/falbo2017patterns/metadata.yaml:8]: Done.

File: models/falbo2017patterns/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+ - patterns
+theme: Class Q - Science
+editorialNote:
+ontologyType:
+ - domain
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/falbo2017patterns/metadata.yaml:13]: Unfortunately, currently there is no better way to classify them. The best approach is to use "domain model" and set as  "learning" or "example". I agree that we have to rethink this for the next version of the ontouml-models-vocabulary.

File: models/machacova2023gym/original-diagrams/posilovna.png
Comment (pedropaulofb) [reply models/machacova2023gym/original-diagrams/posilovna.png:1]: Done. Seems that posilovna (cz) = fitness (en).

File: models/bank-account2013/original-diagrams/Caixa Banco.png
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/bank-account2013/original-diagrams/Caixa Banco.png:1]: Done.

File: models/falbo2017patterns/metadata.yaml (hunk 7 above)
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/falbo2017patterns/metadata.yaml:8]: I agree. Fixed.

File: models/falbo2017patterns/metadata.yaml [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+language: en
+designedForTask:
+ - learning
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/falbo2017patterns/metadata.yaml:16]: I agree. Removed "learning" and kept "example".

File: models/formula-one2023/metadata.yaml [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+ - racing
+ - motorsport
+theme: 
+  - Class G - Geography, Anthropology, and Recreation
+editorialNote:
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - example
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/formula-one2023/metadata.yaml:17]: I am sure it was created to exemplify specific points of the language.

File: models/heirbrant2023boekbazaar/metadata.yaml [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+language: en
+designedForTask:
+ - conceptual clarification
+ - information retrieval
+ - software engineering
+context:
+ - industry
+ - research
+source:
+ - https://lib.ugent.be/catalog/rug01:003144709
```
[the remark this answers is not in the document]
Comment (pedropaulofb) [reply models/heirbrant2023boekbazaar/metadata.yaml:27]: One link points to the PDF and the other to the resource information. I will keep only the resource info.

File: models/heirbrant2023boekbazaar/metadata.yaml (hunk 1 above)
Comment (pedropaulofb) [reply models/heirbrant2023boekbazaar/metadata.yaml:14]: You are right. Fixed.

File: models/heirbrant2023boekbazaar/metadata.yaml (hunk 2 above)
Comment (pedropaulofb) [reply models/heirbrant2023boekbazaar/metadata.yaml:24]: IMHO, the ontology was built for both purposes, as the author opened a company that has a product that uses this ontology. Considering that, should I keep only the research context?

File: models/mereology-examples2013/metadata.yaml (hunk 6 above)
Comment (pedropaulofb) [reply models/mereology-examples2013/metadata.yaml:12]: Ok.

File: models/heirbrant2023boekbazaar/metadata.yaml (hunk 3 above)
Comment (pedropaulofb) [reply models/heirbrant2023boekbazaar/metadata.yaml:10]: Done.

File: models/mereology-examples2013/metadata.yaml (hunk 4 above)
Comment (pedropaulofb) [reply models/mereology-examples2013/metadata.yaml:15]: Done.

**Comment by pedropaulofb** [comment]
This PR is ready for a second review.

File: models/heirbrant2023boekbazaar/metadata.yaml (hunk 2 above)
Comment (tgoprince) [reply models/heirbrant2023boekbazaar/metadata.yaml:24]: I see. Nevermind my comment then.

**Review by tgoprince — APPROVED** [review]

**Pull request merged**